test/metabase/test/data.clj

```diff
@@ -269,5 +269,5 @@
      ;; since the actual group defs are not dynamic, we need with-redefs to change them here
      (with-redefs [perms-group/all-users (#'perms-group/magic-group perms-group/all-users-group-name)
                    perms-group/admin     (#'perms-group/magic-group perms-group/admin-group-name)]
-       (mdb/setup-db!)
+       (mdb/setup-db! :create-sample-content? false) ; skip sample content for speedy tests. this doesn't reflect production
```

Comment:
Oh, I see you do repeat this for a bunch of sites - perhaps just add the relevant comment (for speed or to avoid brittleness) to the non-annotated sites too.

I also feel that this should be a line comment above.

Reply:
I should have added this comment to every site. Did you see a site where this comment is missing?

> this should be a line comment

I added the comment on the end of the line to grep easier to check all sites have the comment. don't care enough to change it

Reply:
Fair. I wish we had a "structural" find-replace + a reliable auto formatter.

Reply:
> Did you see a site where this comment is missing?

For example `cmd.clj` and [this test](https://github.com/metabase/metabase/pull/40753/files#diff-f039ac289d7def5fe424d733725acfdc2cb359516bad4cbdcf2d0078ae2b9a94R34) - I think it's worth going whole hog.

Reply:
Ah thanks for the test, that test should actually have `true` because that one test isn't going to kill test speed. 

For `cmd.clj`, I thought it would be obvious why we don't want the sample content loaded though, based on the context. The purpose of skipping it is not for speed.